CMakeLists.txt

```diff
@@ -20,15 +20,15 @@ link_directories(${binary_dir})
 
 # Enable the latest C++ standard possible
 include(CheckCXXCompilerFlag)
-check_cxx_compiler_flag(--std=c++14 HAVE_FLAG_CXX_14)
```

Comment:
Looking at the `master` I don't think so. This branch has bit rotted for about 3 months. I'll pull it's socks up this week.